rustic-babel.el

```diff
@@ -73,9 +73,18 @@ should be wrapped in which case we will disable rustfmt."
           (toolchain (cond ((eq toolchain-kw-or-string 'nightly) "+nightly")
                            ((eq toolchain-kw-or-string 'beta) "+beta")
                            ((eq toolchain-kw-or-string 'stable) "+stable")
+                           ((or (null rustic-babel-default-toolchain)
+                                (null (string-blank-p rustic-babel-default-toolchain)))
+                            nil)
                            (toolchain-kw-or-string (format "+%s" toolchain-kw-or-string))
                            (t (format "+%s" rustic-babel-default-toolchain))))
-          (params (list "cargo" toolchain "build" "--quiet"))
+          (params (remove nil (list "cargo"
+                                    (unless (or (null toolchain)
+                                                (= 0 (string-blank-p toolchain))
```

Comment:
I belive this will be a problem if `toolchain-kw-or-string` is something like `1.64.0` since toolchain will be `+1.64.0` and evaluating something like this:

```
(= 0 (string-blank-p "+1.64.0"))
```

will result in error:

```
Wrong type argument: number-or-marker-p
```

Reply:
e removed that because that was redundant unnecessarily